core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -4960,6 +4961,28 @@ public void testIfMethodArgument() {
     assertThat(toSql(root, DatabaseProduct.HIVE.getDialect()), isLinux(expectedHive));
   }
 
+  @Test public void testNvlFunctionRelToSql() {
```

Comment:
remove this test from here and add it in sql functions.

Reply:
Done